src/postgraphile/withPostGraphileContext.ts

```diff
@@ -295,9 +299,10 @@ function getSettingsForPgClientTransaction({
     // Try to run `jwt.verify`. If it fails, capture the error and re-throw it
     // as a 403 error because the token is not trustworthy.
     try {
+      const jwtVerificationSecret = jwtPublicKey || jwtSecret;
       // If a JWT token was defined, but a secret was not provided to the server or
       // secret had unsupported type, throw a 403 error.
-      if (!Buffer.isBuffer(jwtSecret) && typeof jwtSecret !== 'string') {
+      if (!Buffer.isBuffer(jwtVerificationSecret) && typeof jwtVerificationSecret !== 'string') {
         // tslint:disable-next-line no-console
         console.error(
           'ERROR: `jwtToken` was provided, but `jwtSecret` was not set to a string or buffer - rejecting request.',
```

Comment:
This error needs fixing to differentiate between `jwtSecret` and `jwtPublicKey`
```suggestion
          `ERROR: '${jwtPublicKey ? 'jwtPublicKey' : 'jwtSecret'}' was not set to a string or buffer - rejecting JWT-authenticated request.`,
```

(This will need to be ran through prettier.)